manifests/overlays/kubeflow/kustomization.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: kubeflow
 resources:
-  - ../../base
+  - ../../base/v1
```

Comment:
@tenzen-y So why you don't want to separate overlays between v1 and v2 version ?

Reply:
I want to prepare `/manifests/v2` and `/manifests/v1` when we create manifests for v2.
Because we want to mitigate the impact for the existing v1 users who use the master branch.

In that case, in this PR, we may want to just add the `/manifests/v2` directory, and then we can keep the existing v1 manifests directory until we introduce the v2 manifests. WDYT?

Reply:
I see, that makes sense, since it will break any users' GitOps/CD that might be configured on the master branch for Training Operator installation.
In that case, are we planning to introduce `/manifests/overlays/v2` in the future ?

Reply:
If you want we can move base + overlays to the v2 folder all together and don't touch v1 at all:
```
/manifests/v2/base/crds
/manifests/v2/base/controller
/manifests/v2/overlays/standalone
/manifests/v2/overlays/kubeflow

Reply:
> If you want we can move base + overlays to the v2 folder all together and don't touch v1 at all:
> 
> ```
> /manifests/v2/base/crds
> /manifests/v2/base/controller
> /manifests/v2/overlays/standalone
> /manifests/v2/overlays/kubeflow
> ```

Yes, I want to select the approach. Let's align this PR with the directory structure.